### PR TITLE
main/libsodium: upgrade to 1.0.18

### PR DESCRIPTION
**Pull request by itoffshore**
* https://github.com/jedisct1/libsodium/blob/master/ChangeLog

### CODE REVIEW

**Comment by itoffshore** [comment]
@maxice8 - added `static` pkg

File: main/libsodium/APKBUILD [hunk 1]
```diff
@@ -22,13 +20,18 @@ build() {
...
 
 check() {
-	cd "$builddir"
 	make check
 }
 
+static() {
+	pkgdesc="libsodium static libraries"
+
+	mkdir -p "$pkgdir"-static/usr/lib
```
Comment (5paceToast) [inline main/libsodium/APKBUILD]: Use `$subpkgdir` instead.
```suggestion
	mkdir -p "$subpkgdir"/usr/lib
```

File: main/libsodium/APKBUILD [hunk 2]
```diff
@@ -22,13 +20,18 @@ build() {
...
+	mv "$pkgdir"/usr/lib/*.a "$pkgdir"-static/usr/lib/
```
Comment (5paceToast) [inline main/libsodium/APKBUILD]: Use `$subpkgdir` instead.
```suggestion
	mv "$pkgdir"/usr/lib/*.a "$subpkgdir"/usr/lib/
```

**Comment by itoffshore** [comment]
@SpaceToast - fixed & rebased

**Pull request closed without merging**